src/main/java/appeng/util/InventoryAdaptor.java

```diff
@@ -51,6 +59,18 @@ public static InventoryAdaptor getAdaptor( final Object te, final EnumFacing d )
 			return null;
 		}
 
+		if ( te instanceof ICapabilityProvider )
+		{
+			ICapabilityProvider capProvider = (ICapabilityProvider) te;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'capProvider' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)